piet-gpu/bin/cli.rs

```diff
@@ -232,7 +232,7 @@ fn main() -> Result<(), Error> {
         let session = Session::new(device);
 
         let mut cmd_buf = session.cmd_buf()?;
-        let query_pool = session.create_query_pool(8)?;
+        let query_pool = session.create_query_pool(12)?;
```

Comment:
```suggestion
        let query_pool = session.create_query_pool(Renderer::QUERY_POOL_SIZE)?;
```

Reply:
missed one :)

Reply:
Gah, I had it in my buffer, just didn't save to the PR. Good catch thanks.